apps/dashboard/src/components/Invoices/List.tsx

```diff
@@ -1,24 +1,137 @@
+        cell: (info) => {
+          const status = info.getValue() as IWithdrawMethod.Type;
+          const value = withdrawMethods[status] as keyof typeof intl;
+          return intl(value);
+        },
+      }),
+      columnHelper.accessor("receiver", {
+        header: intl("dashboard.invoices.receiver"),
+        cell: (info) => info.getValue(),
+      }),
+      columnHelper.accessor("bank", {
+        header: intl("dashboard.invoices.bank"),
+        cell: (info) => info.getValue(),
+      }),
+      columnHelper.accessor("amount", {
+        header: intl("dashboard.invoices.amount"),
+        cell: (info) => formatCurrency(info.getValue()),
+      }),
+      columnHelper.accessor("status", {
+        header: intl("dashboard.invoices.status"),
+        cell: (info) => {
+          const status = info.getValue() as IInvoice.Status;
+          const value = StatusDescriptions[status] as keyof typeof intl;
+          return intl(value);
+        },
+      }),
+      columnHelper.accessor("note", {
+        header: intl("dashboard.invoices.note"),
+        cell: (info) => info.getValue(),
+      }),
+      columnHelper.accessor("attachment", {
+        header: intl("dashboard.invoices.attachment"),
+        cell: (info) => info.getValue(),
+      }),
+      columnHelper.accessor("addressedBy", {
+        header: intl("dashboard.invoices.addressedBy"),
+        cell: (info) => info.getValue(),
+      }),
+      columnHelper.accessor("createdAt", {
+        header: intl("global.createdAt"),
+        cell: (info) => <DateField date={info.row.original.updatedAt} />,
+      }),
+      columnHelper.accessor("updatedAt", {
+        header: intl("global.updatedAt"),
+        cell: (info) => <DateField date={info.row.original.updatedAt} />,
+      }),
+      columnHelper.display({
+        id: "actions",
+        cell: () => (
+          <ActionsMenu
+            actions={[
+              {
+                id: 1,
+                label: intl("global.labels.edit"),
+                onClick() {
+                  // TODO: Dialog to edit invoice
+                  alert("edit");
+                },
+              },
+              {
+                id: 2,
+                label: intl("global.labels.delete"),
+                danger: true,
+                onClick() {
+                  // TODO: Dialog to remove invoice
+                  alert("Delete Invoice!!!");
+                },
+              },
+            ]}
+          />
+        ),
+      }),
+    ],
+    [columnHelper, intl, invoicesList]
+  );
 
   return (
     <div className="w-full">
-      <Timeline timeline={timeline} />
+      <Table
+        columns={columns}
+        data={invoicesList || []}
+        goto={() => {}}
+        prev={query.fetchPreviousPage}
+        next={query.fetchNextPage}
+        fetching={query.isFetching}
+        loading={query.isLoading}
+        totalPages={query.data?.pages.length || 1}
+        page={1}
+      />
```

Comment:
The dialog below should be rendered here as well. 

https://github.com/litespace-org/litespace/blob/a66d1dee9b39eecdc785fd85f1c6b3e47ed5e216/apps/dashboard/src/components/Invoices/Invoice.tsx#L150-L160